bundles/binding/org.openhab.binding.caldav-personal/README.md

```diff
@@ -82,29 +82,34 @@ Depending on your rule implementation, it is possible to use event entries like
 # CalDAV Personal
 Binding file: org.openhab.binding.caldav-personal-version.jar
 
-* Used to detect presence through calendar events.
-* Used to show upcoming/active events in openhab.
+* Used to detect presence through calendar events
+* Used to show upcoming/active events in openhab
 
 ### openhab.cfg
-* `caldavPersonal:usedCalendars=<calendar-id>` (multiple calendars can be seperated by commas)
-* `caldavPersonal:homeIdentifiers=<values seperated by commans>` (if one of these identifiers can be found inside the place of the event, this event will not be used for presence)
+* `caldavPersonal:usedCalendars=<calendar-id>` (multiple calendars can be separated by commas)
+* `caldavPersonal:homeIdentifiers=<values separated by commans>` (if one of these identifiers can be found inside the place of the event, this event will not be used for presence)
```

Comment:
Can you fix this `commans` typo?

Reply:
I've been working from the web interface and I'd rather submit another PR after this one is merged. OK?

Reply:
Not sure it's worth that, but if you want, I'm ok with it.